pseudomonas_analysis/2_identify_generic_genes_pathways_pao1.ipynb

```diff
@@ -0,0 +1,1764 @@
+       "PA2260  PA2260            0.034741       5387.0           0.953635   \n",
+       "PA3915  PA3915            0.000920       5535.0           2.065593   \n",
+       "PA5168  PA5168            0.005137       5475.0           1.292737   \n",
+       "\n",
+       "        logFC (Real)  Median adj p-value (simulated)  Rank (simulated)  \\\n",
+       "PA2195     -1.549834                        0.733340            3092.0   \n",
+       "PA4726     -4.530917                        0.554727            4636.5   \n",
+       "PA2260      0.953635                        0.741730            1713.0   \n",
+       "PA3915     -2.065593                        0.391412            5308.0   \n",
+       "PA5168     -1.292737                        0.707260            3344.5   \n",
+       "\n",
+       "        Mean abs(logFC) (simulated)  Std deviation (simulated)  \\\n",
+       "PA2195                      0.12754                   0.077098   \n",
+       "PA4726                      0.24804                   0.242961   \n",
+       "PA2260                      0.08960                   0.050718   \n",
+       "PA3915                      0.28870                   0.109285   \n",
+       "PA5168                      0.13870                   0.075171   \n",
+       "\n",
+       "        Number of experiments (simulated)    Z score Gene Name  \n",
+       "PA2195                                 25  18.447771      hcnC  \n",
+       "PA4726                                 25  17.627806      cbrB  \n",
+       "PA2260                                 25  17.036139       NaN  \n",
+       "PA3915                                 25  16.259220     moaB1  \n",
+       "PA5168                                 25  15.352146      dctQ  "
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Add gene name as column to summary dataframe\n",
+    "summary_gene_ranks = ranking.add_pseudomonas_gene_name_col(summary_gene_ranks, base_dir)\n",
+    "summary_gene_ranks.sort_values(by=\"Z score\", ascending=False).head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Gene ID                              False\n",
+       "Adj P-value (Real)                   False\n",
+       "Rank (Real)                          False\n",
+       "abs(logFC) (Real)                    False\n",
+       "logFC (Real)                         False\n",
+       "Median adj p-value (simulated)       False\n",
+       "Rank (simulated)                     False\n",
+       "Mean abs(logFC) (simulated)          False\n",
+       "Std deviation (simulated)            False\n",
+       "Number of experiments (simulated)    False\n",
+       "Z score                              False\n",
+       "Gene Name                             True\n",
```

Comment:
Is it okay for the Gene Name column to have NaNs?

Reply:
Yes this is ok. Some PA### ids don't have a gene name associated.